Formula/haproxy.rb

```diff
@@ -25,6 +26,12 @@ def install
       ADDLIB=-lcrypto
     ]
 
+    if build.with?("lua")
+      lua = Formula["lua"]
+      args << "USE_LUA=1"
```

Comment:
you should still use the `LUA_INC=#{lua.opt_include}"` so this will work in non-/usr/local prefixes

Reply:
@ilovezfs done